### PR TITLE
ENCD-4431 change report file download name

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: src/encoded/batch_download.py [hunk 1]
```diff
@@ -449,7 +449,7 @@ def report_download(context, request):
     # Make sure we get all results
     request.GET['limit'] = 'all'
 
-    type = types[0]
+    type = types[0].lower()
```
[the remark this answers is not in the document]
Comment (caseylitton) [reply src/encoded/batch_download.py]: never mind that was preexisting.  Don't change it.

[the remark this answers is not in the document]
Comment (gabdank) [reply src/encoded/batch_download.py]: kk

[the remark this answers is not in the document]
Comment (gabdank) [reply src/encoded/batch_download.py]: @caseylitton so is it code QA'ed? can the ticket be moved to QA ready?

**Comment by keenangraham** [comment]
You'll have to fix this test for sure: 

```
src/encoded/tests/test_batch_download.py::test_batch_download_report_download FAILED
_____________________ test_batch_download_report_download ______________________
testapp = <webtest.app.TestApp object at 0x7f0005dbe9e8>, workbook = None
    def test_batch_download_report_download(testapp, workbook):
        res = testapp.get('/report.tsv?type=Experiment&sort=accession')
        assert res.headers['content-type'] == 'text/tsv; charset=UTF-8'
        disposition = res.headers['content-disposition']
>       assert disposition.startswith('attachment;filename="Experiment Report') and disposition.endswith('.tsv"')
E       assert (<built-in method startswith of str object at 0x7f0004c70880>('attachment;filename="Experiment Report'))
E        +  where <built-in method startswith of str object at 0x7f0004c70880> = 'attachment;filename="experiment_report_2019_2_11_21h_5m.tsv"'.startswith
```

There's also a number of these type of errors:
```

INFO:encoded:Loaded 12 of 12 analysis_step (phase 2). CREATED: 0, UPDATED: 12, SKIPPED: 0, ERRORS: 0
INFO:encoded:Loaded 25 of 25 file (phase 2). CREATED: 0, UPDATED: 25, SKIPPED: 0, ERRORS: 0
WARNING:snovault.elasticsearch.es_index_listener:running 1549919047976223 with 1032
Error rendering /4f2ffcd8-8673-474b-8d79-f0fbe6777e08/@@index-data
Traceback (most recent call last):
  File "/home/travis/build/ENCODE-DCC/encoded/develop/snovault/src/snovault/embed.py", line 80, in _embed
    result = request.invoke_subrequest(subreq)
  File "/home/travis/build/ENCODE-DCC/encoded/eggs/pyramid-1.6a2-py3.4.egg/pyramid/router.py", line 198, in invoke_subrequest
    response = handle_request(request)
  File "/home/travis/build/ENCODE-DCC/encoded/eggs/pyramid-1.6a2-py3.4.egg/pyramid/router.py", line 162, in handle_request
    raise HTTPNotFound(msg)
pyramid.httpexceptions.HTTPNotFound: /4f2ffcd8-8673-474b-8d79-f0fbe6777e08/@@index-data/
```

**Pull request merged**